### PR TITLE
🚀 Login: Initialized a new empty app

### PR DESCRIPTION
**Pull request by amittcohen**
Initialization of an empty app for login.
Can be viewed under http://localhost:8000/login/ (minimal page, only to validate the routing)

### CODE REVIEW

**Review by DeanBiton — APPROVED** [review]
LGTM

**Review by taljacob2 — APPROVED** [review]
LGTM!

If you want to add tests you can take a look at [4cf4d57](https://github.com/redhat-beyond/JobSeeker/pull/43/commits/4cf4d5762c6aa801dab4693f09e6a0472c377646),
and change the names from *"chat"* to *"login"*.

File: login/models.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# from django.db import models
```
Comment (Yarboa) [inline login/models.py:1]: Is there any data exist in addition to django user model?

**Review by Yarboa — COMMENTED** [review]
For the 2nd please concentrate in model and not UI

This PR should be relevant if you need extra data to extend django user model

File: login/models.py (hunk 1 above)
Comment (amittcohen) [reply login/models.py:1]: No additional data models.

File: login/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
+
```
Comment (Yarboa) [inline login/tests.py:2]: PR without any tests? 
Please add some login check

**Comment by Yarboa** [comment]
Please rebase and push

**Comment by amittcohen** [comment]
> Please rebase and push

@Yarboa Done

File: app/settings.py [hunk 3]
```diff
@@ -40,6 +40,7 @@
     'jobseeker.apps.JobseekerConfig',
     'django.contrib.staticfiles',
     'login.apps.LoginConfig'
```
Comment (Yarboa) [inline app/settings.py:42]: I think this is failing your test 
```
'login.apps.LoginConfig'
'job_board.apps.JobBoardConfig',
```
search for the comma

File: login/tests.py (hunk 2 above)
Comment (Yarboa) [reply login/tests.py:2]: Please consider to add here tests
If you have login page at lease add test to verify it is working

Comment (amittcohen) [reply login/tests.py:2]: This is a pr only for initialization of the login app, without any login form.
I will add the different pr with the login form, and will add tests to it.

Comment (taljacob2) [reply login/tests.py:2]: @amittcohen You can add this simple test: https://github.com/redhat-beyond/JobSeeker/pull/55#pullrequestreview-920744972

Comment (Yarboa) [reply login/tests.py:2]: @amittcohen 
Please add a test, it is mandatory for Merge
As per taljacobs's sample

**Comment by Yarboa** [comment]
@amittcohen
Please add a test, it is mandatory for Merge
As per taljacobs's sample

**Comment by amittcohen** [comment]
> @amittcohen Please add a test, it is mandatory for Merge As per taljacobs's sample

@Yarboa Done. added the test and also a simple login form.

**Comment by taljacob2** [comment]
Looks great!
I'm posting here a screenshot for the http://localhost:8000/login/ page you have made:

image

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**